pkg/rtc/signalhandler.go

```diff
@@ -57,8 +57,6 @@ func HandleParticipantSignal(room types.Room, participant types.LocalParticipant
 				pLogger.Errorw("failed to update subscribed track settings", err, "trackID", sid)
 				continue
 			}
-
-			pLogger.Infow("updated subscribed track settings", "trackID", sid, "settings", msg.TrackSetting)
 		}
```

Comment:
Probably leave this in at Infow till we have signal deduper full confidence?

Reply:
I don't believe signal deduper is currently handling adaptive stream messages. These are being logged in SubscribedTrack when there's an actual change in layer or visibility. I thought that'd be sufficient. wdyt?

Reply:
Signal deduper does handle UodateTtackSettings messages. We get a lot of those duplicates too. As long as it is logged in SubscribedTrack, it should be fine.